lib/Driver.cpp

```diff
@@ -63,25 +63,34 @@ std::vector<std::unique_ptr<TestResult>> Driver::Run() {
 
   for (auto &Test : Finder.findTests(Ctx)) {
     auto ObjectFiles = AllObjectFiles();
-    ExecutionResult ExecResult = Runner.runTest(Test.get(), ObjectFiles);
+
+    ExecutionResult ExecResult = Sandbox.run([&](ExecutionResult *SharedResult){
+      *SharedResult = Runner.runTest(Test.get(), ObjectFiles);
+    });
+
     auto BorrowedTest = Test.get();
     auto Result = make_unique<TestResult>(ExecResult, std::move(Test));
 
     for (auto Testee : Finder.findTestees(BorrowedTest, Ctx)) {
       auto ObjectFiles = AllButOne(Testee->getParent());
       for (auto &MutationPoint : Finder.findMutationPoints(MutationOperators, *Testee)) {
+        ExecutionResult R = Sandbox.run([&](ExecutionResult *SharedResult){
```

Comment:
All the setup work already done at this point. We just deferring mutation and mutant execution to a child process to gain implicit module copy (mentioned above) and to prevent ourselves from hanging/crashing.